kartothek/core/common_metadata.py

```diff
@@ -616,6 +638,15 @@ def validate_compatible(schemas, ignore_pandas=False):
             current, null_columns
         ).remove_metadata()
 
+        def _fmt_origin(origin):
+            origin = sorted(origin)
+            # cut origin after a certain length since dask & others cut overlong exception messages
+            max_elements = 5
+            if len(origin) > max_elements:
```

Comment:
I know this is nitpicking but considering that dask cuts off at 1k chars, see https://github.com/dask/distributed/blob/6e0c0a6b90b1d3c3f686f0c968e9cf3d0c354413/distributed/core.py#L964
showing 5 origins may be even too much and we should rather cut regarding the length of the strings. Consider a realistic example (table name, two partition cols and a hex UUID)
```
table_name/a_string_column=a_long_name/a_date_column=2018-09-12/ef1aa9f0ce3e49e496ea93e889e812bc
```
we have 96 chars for a single origin. With 5 origins times two (reference and schema), the diff is completely cut off already.

Maybe we should rather do smth like
```python
lenghts = list(map(len, origins))
if sum(lengths) > 200:
# cut off but ensure there is at least one
else:
# return everything
```

On a side note: Should we log the entire exception? Good idea / horrible idea? I don't have much experience with loggers who receive *big* log messages